packages/components/src/dateInput/DateInput.js

```diff
@@ -2,58 +2,107 @@
 
 import React from 'react';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
-import { format, setMonth } from 'date-fns';
 import { View } from 'react-native';
 import {
   StyleSheet,
   Picker,
   Text,
   TextInput,
 } from '@kiwicom/universal-components';
+import { setMonth, getDate, getMonth, getYear, format } from 'date-fns';
+import { MAX_YEAR_LENGTH, MAX_DAY_LENGTH } from '@kiwicom/margarita-config';
+
+import { composeDateFromStrings } from './helpers';
 
 type Props = {|
-  +onDateChange: (string, string) => void,
-  +date: {
-    +day: string,
-    +month: string,
-    +year: string,
-  },
-  +errors: {
-    +day: string,
-    +year: string,
-  },
+  +onDateChange: Date => void,
+  +date: ?Date,
+  +editMode: boolean,
+  +getDateRequest: boolean,
```

Comment:
It's not clear what it does from the name if it's boolean. Boolean should have that `is|should|has...` prefix

Reply:
Agree! What about then `isEditMode` and `isDateRequested`?